rm/rm-server/src/test/java/org/ow2/proactive/resourcemanager/core/RMCoreTest.java

```diff
@@ -622,69 +618,6 @@ private void testLockNodeState(NodeState nodeState) {
         assertThat(freeNodes).isEmpty();
     }
 
-    @Test
-    public void testInternalLockNodeWithNodeNotLocked() {
-        verify(dbManager, never()).createLockEntryOrUpdate(anyString(), any(RMDBManager.NodeLockUpdateAction.class));
```

Comment:
Is there at least a set of functional tests or unit tests which check the locks are restored properly after the changes ?

Reply:
Only a unit test for now. I'll add lock checks in the recovery functional test today.
Also a system test tests lock recovery but its logic needs to be changed since it kills the nodes before restarting.